src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -141,7 +141,7 @@ private void initializeProducer() throws Exception {
     producerProps.putAll(_producerPropsOverride);
 
     _producer = (KMBaseProducer) Class.forName(_producerClassName).getConstructor(Properties.class).newInstance(producerProps);
-    LOG.info("Producer is initialized.");
+    LOG.info(_name + ": Producer is initialized.");
```

Comment:
I noticed that we already have `LOG.info(_name + ": produce service is initialized.")` in the constructor of `ProduceService`. Maybe we can remove this `LOG.INFO` statement and update the `LOG.info("New partitions added to monitoring.")` in `NewPartitionHandler.run()` to include `_name`.